src/components/specific/files/files-manager/FilesManager.vue

```diff
@@ -28,17 +28,19 @@
             clear
           />
           <BIMDataButton
+            :disabled="!userVisas"
             class="files-manager__actions__visa"
             color="primary"
             fill
             radius
             @click="openVisaManager"
           >
             <span class="files-manager__actions__visa__content">
-              <span class="files-manager__actions__visa__content__counter">{{
-                userVisas.toValidateVisas.length +
-                  userVisas.createdVisas.length || ""
-              }}</span>
+              <template v-if="userVisasCounter > 0">
```

Comment:
```suggestion
              <template v-if="userVisas.toValidateVisas.length || userVisas.createdVisas.length">
```
This way, you don't need to create `userVisasCounter` at all